benchmarks/perception/a3_stereo_image_proc/include/a3_stereo_image_proc/disparity_output_component.hpp

```diff
@@ -0,0 +1,46 @@
+#ifndef ROBOTPERF_DISPARITY_OUTPUT_COMPONENT_HPP_
+#define ROBOTPERF_DISPARITY_OUTPUT_COMPONENT_HPP_
+
+#include <rclcpp/rclcpp.hpp>
+#include <image_transport/image_transport.hpp>
+#include <ament_index_cpp/get_resource.hpp>
+#include <stereo_msgs/msg/disparity_image.hpp>
+
+#include <cstring>
+#include <memory>
+#include <sstream>
+#include <string>
+#include <vector>
+#include <thread>
```

Comment:
From the context below, you don't need any of these headers included in this interface definition file.  Consider moving to the source implementation if they are really needed.

Reply:
thanks for pointing this out, yep, they aren't needed. removed!